scripts/install-chef-suse.sh

```diff
@@ -661,11 +615,11 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
     #        Additional work (e.g. on the autoyast profile) is required to make
     #        those repos available to any client nodes.
     if [ $CROWBAR_FROM_GIT = "ibs" ]; then
-        add_ibs_repo http://dist.suse.de/install/SLP/SLES-11-SP3-LATEST/x86_64/DVD1 sp3
-        add_ibs_repo http://dist.suse.de/install/SLP/SLE-11-SP3-SDK-LATEST/x86_64/DVD1/ sdk-sp3
-        add_ibs_repo http://dist.suse.de/ibs/SUSE:/SLE-11-SP3:/GA/standard/ sp3-ga
-        add_ibs_repo http://dist.suse.de/ibs/SUSE:/SLE-11-SP3:/Update/standard/ sp3-update
-        add_ibs_repo http://dist.suse.de/ibs/Devel:/Cloud:/3/SLE_11_SP3/ cloud
+        add_ibs_repo http://dist.suse.de/install/SLP/SLES-12-LATEST/x86_64/DVD1 sp3
+        add_ibs_repo http://dist.suse.de/install/SLP/SLE-12-SDK-LATEST/x86_64/DVD1/ sdk-sp3
+        add_ibs_repo http://dist.suse.de/ibs/SUSE:/SLE-12:/GA/standard/ sp3-ga
+        add_ibs_repo http://dist.suse.de/ibs/SUSE:/SLE-12:/Update/standard/ sp3-update
+        add_ibs_repo http://dist.suse.de/ibs/Devel:/Cloud:/6/SLE_12/ cloud
```

Comment:
It's the `CROWBAR_FROM_GIT` codepath which seems to be broken anyway. There was still `Cloud 3` used...